common/crypto_utils/avalon_crypto_utils/keys.py

```diff
@@ -35,13 +36,13 @@ def __init__(self, verifying_key, encryption_key):
         :param verifying_key: PEM encoded ECDSA verifying key
         :param encryption_key: PEM encoded RSA encryption key
         """
-        self._verifying_key = crypto.SIG_PublicKey(verifying_key)
-        self._encryption_key = crypto.PKENC_PublicKey(encryption_key)
+        self._verifying_key = VerifyingKey.from_pem(verifying_key)
+        self._encryption_key = RSA.importKey(encryption_key)
 
     # -------------------------------------------------------
     @property
     def identity(self):
-        return self._verifying_key.Serialize()
+        return self._verifying_key.to_string()
```

Comment:
Does this give PEM formatted string. Else you can use `self._verifying_key.to_pem()`

Reply:
Done